models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
+  family : string
+  given : string
+  suffix : string
+  phone : {
+    value : string
+    use : enum (home, work, temp, old, mobile)
+  }
+  email : array of strings
+  gender : enum (male, female, other, unknown)
+  birthdate : string
+  race : enum (American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Black or African-American, White, Other Race)
+  ethnicity: enum (Hispanic, Not Hispanic)
+  permissionToFollowup: boolean
+  address: {
+    line: string
+    city: string
+    state: string
+    postalCode: string
+    country: string
+  }
```

Comment:
What about patient photo?

Reply:
I am looking into storing the patient photo, but it's not a priority atm. I'll keep this in mind